src/schemes/boundary/dummy_particles/dummy_particles.jl

```diff
@@ -378,68 +415,115 @@ function compute_pressure!(boundary_model, ::Union{PressureMirroring, PressureZe
     return boundary_model
 end
 
-@inline function adami_pressure_extrapolation_neighbor!(boundary_model, system,
-                                                        neighbor_system, system_coords,
-                                                        neighbor_coords, v_neighbor_system,
-                                                        neighborhood_search)
+@inline function dynamic_pressure(density_neighbor, v, v_neighbor_system, particle,
+                                  neighbor, system::BoundarySystem)
+    if system.ismoving[]
+        return 0.5 * density_neighbor *
+               norm(current_velocity(v, system, particle) -
+                    v_neighbor_system[1:ndims(system), neighbor])^2
+    end
+    return 0.0
+end
+
+@inline function dynamic_pressure(density_neighbor, v, v_neighbor_system, particle,
+                                  neighbor, system::SolidSystem)
+    return 0.5 * density_neighbor *
+           norm(current_velocity(v, system, particle) -
+                v_neighbor_system[1:ndims(system), neighbor])^2
+end
+
+@inline function bnd_pressure_extrapolation_neighbor!(boundary_model, system,
```

Comment:
```suggestion
@inline function boundary_pressure_extrapolation_neighbor!(boundary_model, system,
```
I know it's a long name, but I'd prefer the full version for clarity and consistency reasons.